assets/state-container-toolkit/0500_daemonset.yaml

```diff
@@ -67,6 +71,8 @@ spec:
           value: "management.nvidia.com/gpu"
         - name: NVIDIA_VISIBLE_DEVICES
           value: "void"
+        - name: TOOLKIT_PID_FILE
```

Comment:
For reference. This requires https://github.com/NVIDIA/nvidia-container-toolkit/pull/544